.github/workflows/AWS-auth.yml

```diff
@@ -0,0 +1,68 @@
+name: AWS Auth + Welcome Email
+
+on:
+  push:
+    branches:
+      - 'test_userpool_creation'
```

Comment:
This needs to be changed. After the merge, we no longer want to run this only on the `test_userpool_creation` branch